NativeApp/include/AppBase.hpp

```diff
@@ -70,6 +70,19 @@ class AppBase
         Error
     };
 
+    /// Flags to handle default hotkeys
+    enum HOT_KEY_FLAGS : Uint32
+    {
+        /// App doesn't react on default hotkeys
+        HOT_KEY_FLAG_NONE = 0,
+
+        /// App will exit by pressing Escape
+        HOT_KEY_FLAG_ALLOW_EXIT_ON_ESC = 1,
+
+        /// App wil change fullscreen mode on Shift+Enter (only in UWP)
+        HOT_KEY_FLAG_ALLOW_FULL_SCREEN_SWITCH = 1 << 1
+    };
```

Comment:
Please add the `HOT_KEY_FLAG_ALL = ~0u` value.

Please add the `DEFINE_FLAG_ENUM_OPERATORS` macro, see example [here](DEFINE_FLAG_ENUM_OPERATORS).